101-sql-with-failover-group/metadata.json

```diff
@@ -0,0 +1,7 @@
+{
+  "itemDisplayName": "Create Azure SQL Servers and Database with Failover Group",
+  "description": "Creates two Azure SQL servers, a database, and a failover group. Template originally authored by John Downs.",
```

Comment:
remove the authored by comments since it pulled from metadata in various experiences